bindings/src/main/scala/chrome/contextMenus/ContextMenus.scala

```diff
@@ -0,0 +1,34 @@
+package chrome.contextMenus
+
+import chrome.ChromeAPI
+import chrome.contextMenus.bindings.{CreateProperties, MenuInfo, UpdateProperties}
+import chrome.permissions.APIPermission
+import chrome.tabs.bindings.Tab
+
+import scala.scalajs.js
+import scala.scalajs.js.|
+import scalajs.js.JSConverters._
+
+/**
+  * Created by antonkulaga on 07/06/16.
+  */
+object ContextMenus  extends ChromeAPI
+{
+  val ContextMenus = APIPermission("contextMenus", "permission to add items to content menus")
```

Comment:
all the `APIPermissions` are currently in `APIPermission` permission companion object.